### PR TITLE
LC TPM DMA Enable bit

### PR DESCRIPTION
**Pull request by kbob**
Hi, Paul.

The Kinetis-L chip in Teensy LC has an additional flag, DMA enable, in the TPMx_SC (or FTMx_SC) registers.  This patch adds a definition for that bit.

Thanks.

### CODE REVIEW

File: teensy3/DMAChannel.h [hunk 1]
```diff
@@ -791,7 +791,7 @@ class DMABaseClass {
 		d = (dcr >> 17) & 3;
 		if (s == 0 || d == 0) n = 2;
 		else if (s == 2 || d == 2) n = 1;
-		CFG->DSR_BCR = len >> n;
+		CFG->DSR_BCR = len << n;
```
Comment (PaulStoffregen) [inline teensy3/DMAChannel.h]: why is this changed?

**Comment by kbob** [comment]
I take it you figured it out, since you merged it.

The DSR_BCR register is Byte Count Register.  transferCount() is called
with a word count (either long, short or byte).  To convert words to bytes,
the quantity needs to be shifted left, not right.

On Wed, May 20, 2015 at 9:43 AM, Paul Stoffregen <notifications@github.com>
wrote:

> Merged #75 <https://github.com/PaulStoffregen/cores/pull/75>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PaulStoffregen/cores/pull/75#event-309730659>.
>



-- 
Bob Miller                              K<bob>
                                        kbob@jogger-egg.com

**Comment by PaulStoffregen** [comment]
yup, good catch
